R/geom-smooth.r

```diff
@@ -1,22 +1,22 @@
 #' Smoothed conditional means
 #'
 #' Aids the eye in seeing patterns in the presence of overplotting.
-#' `geom_smooth` and `stat_smooth` are effectively aliases: they
-#' both use the same arguments. Use `geom_smooth` unless you want to
+#' `geom_smooth()` and `stat_smooth()` are effectively aliases: they
+#' both use the same arguments. Use `geom_smooth()` unless you want to
```

Comment:
Maybe state this in the positive? Use `stat_smooth()` if you want to display the results with a non-standard geom.